instructions/bundling-scripts/zcs-bundle.sh

```diff
@@ -41,7 +41,6 @@ cp -f ${repoDir}/zm-build/rpmconf/Install/Util/globals.sh
 cp -f ${repoDir}/zm-build/rpmconf/Install/Util/modules/getconfig.sh                     ${ZCS_REL}/util/modules
 cp -f ${repoDir}/zm-build/rpmconf/Install/Util/modules/packages.sh                      ${ZCS_REL}/util/modules
 cp -f ${repoDir}/zm-build/rpmconf/Install/Util/modules/postinstall.sh                   ${ZCS_REL}/util/modules
-cp -f ${repoDir}/zm-build/rpmconf/Install/Util/modules/install_rabbitmq.sh              ${ZCS_REL}/util/modules
```

Comment:
Why we are removing this? are we handling rabbitmq differently?

Reply:
yes